Gruntfile.js

```diff
@@ -84,6 +83,32 @@ module.exports = function(grunt) {
         command: './run_python_tests.sh'
       },
     },
+    
+    'grunt-chrome-build' : {
+      apprtc: {
+        options: {
+          buildDir: 'build/chrome-app',
+          zipFile: 'build/chrome-app/apprtc.zip',
+          chromeBinary: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome',
+          keyFile: 'apprtc.pem'
```

Comment:
chromeBinary/keyFile are not currently used, they will be part of step 6 once we decide the right way to implement it.